src/Vcs/VersionControlSystem.php

```diff
@@ -17,4 +17,6 @@ public function pushVersion(string $version): void;
      * @return Commit[]
      */
     public function getCommitsSinceLastVersion(?string $pattern = null): array;
+
+    public function getTagForVersion(string $version): string;
```

Comment:
Are tags a concept that can be used in all version control systems, or are they specific to Git?

Reply:
Seems to be a concept in Mercurial and SVN as well.